src/sentry/search/events/constants.py

```diff
@@ -83,6 +83,6 @@
     ">=": "<",
     "IN": "NOT IN",
 }
-OPERATOR_TO_DJANGO = {">=": "gte", "<=": "lte", ">": "gt", "<": "lt"}
+OPERATOR_TO_DJANGO = {">=": "gte", "<=": "lte", ">": "gt", "<": "lt", "=": "exact"}
```

Comment:
How did equal work before? Why didn't this already exist?

Reply:
This is only used in a small number of places, mostly https://github.com/getsentry/sentry/blob/6d1f63bba9effa69ce4dba088bc86048da41eae8/src/sentry/search/snuba/backend.py#L283.

Adding `=` here just makes it so I don't need to special case equality when using django filters.